aep/general/0151/aep.md.j2

```diff
@@ -75,10 +75,23 @@ but is not obligated to do so:
 
 - Resources that accept multiple parallel requests **may** place them in a
   queue rather than work on the requests simultaneously.
-- Resource that does not permit multiple requests in parallel (denying any new
-  request until the one that is in progress finishes) **must** return
-  `409 Conflict` if a user attempts a parallel request, and include an error
-  message explaining the situation.
+
+{% tab proto %}
+
+- A Resource that does not permit multiple requests in parallel (denying any new
```

Comment:
"multiple requests" is inaccurate. Multiple GET requests should be fine.

Reply:
LROs will apply to more than GET though - for example multiple updates may be requested.

Reply:
applied the suggestions! thanks.